python/sparta.py

```diff
@@ -56,33 +56,36 @@ def close(self):
     self.spa = None
 
   def file(self,file):
-    self.lib.sparta_file(self.spa,file)
+    b_file = file.encode('utf-8')
+    self.lib.sparta_file(self.spa,c_char_p(b_file))
```

Comment:
The `c_char_p` is not needed on my end.

Reply:
thanks @stefanfechter - I will try this w/out on my box

Reply:
@stefanfechter you're right - the c_chap_p() cast is not needed - so I removed those, leaving the encoding function there - thanks!